tests/vm_test.go

```diff
@@ -1653,16 +1654,32 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 						newVMI, err := virtClient.VirtualMachineInstance(newVM.Namespace).Get(newVM.Name, &k8smetav1.GetOptions{})
 						Expect(err).ToNot(HaveOccurred())
 						return newVMI.Status.Phase == v1.Failed
-					}, 400*time.Second, 1*time.Second).Should(BeTrue())
+					}, 480*time.Second, 1*time.Second).Should(BeTrue())
 
-					// Virt launcher should be in the Running phase if the Annotation v1.KeepLauncherAfterFailureAnnotation is set to true
-					By(fmt.Sprintf("Verify that the VMI VirtLauncher Pod is in the expected state as %v", podPhase))
+					// The containerStatus of the compute container of the virt-launcher pod should be in the Running state
+					// if the Annotation v1.KeepLauncherAfterFailureAnnotation is set to true
+					By(fmt.Sprintf("Verify that the compute container of the virt-launcher pod is in the running state"))
 					newVMI, _ := virtClient.VirtualMachineInstance(newVM.Namespace).Get(newVM.Name, &k8smetav1.GetOptions{})
 					launcherPod := libvmi.GetPodByVirtualMachineInstance(newVMI, newVM.Namespace)
-					Expect(launcherPod.Status.Phase).To(Equal(podPhase))
+					keepLauncherBool, _ := strconv.ParseBool(keepLauncher)
+
+					if !keepLauncherBool {
```

Comment:
nit: Can we do `if keepLauncherBool` instead and swap blocks so the `else` case is not a double-negative?

Reply:
Uploaded changes to comply with the comment.